docs/containers-storage.conf.5.md

```diff
@@ -138,6 +138,9 @@ The `storage.options.thinpool` table supports the following options for the `dev
     6: LogLevelInfo
     7: LogLevelDebug
 
+**metadata_size**="1k"
+  metadata_size is used to set the `pvcreate --metadatasize` options when creating thin devices.
```

Comment:
is 1k the only value accepted?  I was expecting `**metadata_size**=""` here with a description of the accepted values in the text.

Reply:
```suggestion
Specifies a custom metadata_size to be used to set the `pvcreate --metadatasize` option when creating thin devices. (default: 1k)
```

Reply:
Nope any value accepted by pvcreate, but the default is 1k.  Or at least this is what I am told.